airbyte-integrations/connectors/source-marketo/source_marketo/utils.py

```diff
@@ -37,8 +37,15 @@ def clean_string(string: str) -> str:
     "updatedat" -> "updatedat"
     """
 
-    abbreviations = ("URL", "GUID", "IP")
-    if any(map(lambda w: w in string, abbreviations)):
+    fix = {
```

Comment:
can we have unit tests?

Reply:
I have added more cases for this unit_test 'test_clean_string'